### PR TITLE
Build CAPI Ubuntu image for GCE

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

### CODE REVIEW

File: images/capi/packer/gce/packer.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+  "variables": {
+    "project_id": "{{env `GCP_PROJECT_ID`}}",
+    "account_file": "{{env `GOOGLE_APPLICATION_CREDENTIALS`}}",
```
Comment (detiber) [inline images/capi/packer/gce/packer.json]: Shouldn't these be picked up by the environment anyway if not specified?

File: images/capi/packer/gce/packer.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    "existing_ansible_ssh_args": "{{env `ANSIBLE_SSH_ARGS`}}"
+  },
+  "builders": [
+    {
+      "name": "ubuntu-1804",
+      "type": "googlecompute",
+      "project_id": "{{ user `project_id` }}",
+      "account_file": "{{ user `account_file` }}",
+      "machine_type": "n1-standard-1",
+      "source_image_family": "ubuntu-1804-lts",
```
Comment (detiber) [inline images/capi/packer/gce/packer.json]: Should this be exposed as a variable to allow users to override it?

File: images/capi/packer/gce/packer.json (hunk 1 above)
Comment (vincepri) [reply images/capi/packer/gce/packer.json]: They weren't, at least in testing :/

File: images/capi/packer/gce/packer.json (hunk 2 above)
Comment (vincepri) [reply images/capi/packer/gce/packer.json]: Is there a way to default it to this, unless the user specifies it?

Comment (detiber) [reply images/capi/packer/gce/packer.json]: Yes, setting it as the default value in the `variables` definition.

File: images/capi/packer/gce/packer.json (hunk 1 above)
Comment (detiber) [reply images/capi/packer/gce/packer.json]: It should according to the docs: https://www.packer.io/docs/builders/googlecompute.html#precedence-of-authentication-methods

Comment (vincepri) [reply images/capi/packer/gce/packer.json]: Nice, need more coffee today 😄

File: images/capi/packer/gce/packer.json (hunk 2 above)
Comment (vincepri) [reply images/capi/packer/gce/packer.json]: Fixed!

**Pull request merged**